### PR TITLE
Release beta 4

### PR DESCRIPTION
**Pull request by rprouse**
Putting this up for review as I package beta 4. Assuming all is good, I will merge once the beta 4 release is ready, but in the meantime, please review. Mainly packaging details, plus fixed build warnings.

### CODE REVIEW

File: src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj [hunk 1]
```diff
@@ -42,12 +42,6 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\packages\Mono.Cecil.0.9.6.1\lib\net20\Mono.Cecil.dll</HintPath>
     </Reference>
-    <Reference Include="Mono.Cecil.Mdb">
```
Comment (rprouse) [inline src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj]: These assemblies weren't in the NuGet package that I restored causing warnings. They don't seem to be used.

File: Rebracer.xml [hunk 2]
```diff
@@ -11,9 +11,14 @@
       <ToolsOptionsSubCategory name="CSharp-Specific">
         <PropertyValue name="AutoComment">1</PropertyValue>
         <PropertyValue name="BringUpOnIdentifier">1</PropertyValue>
+        <PropertyValue name="ClosedFileDiagnostics">1</PropertyValue>
         <PropertyValue name="CodeDefinitionWindow_DocumentationComment_IndentBase">1</PropertyValue>
         <PropertyValue name="CodeDefinitionWindow_DocumentationComment_IndentOffset">2</PropertyValue>
         <PropertyValue name="CodeDefinitionWindow_DocumentationComment_WrapLength">80</PropertyValue>
+        <PropertyValue name="DisplayLineSeparators">0</PropertyValue>
```
Comment (rprouse) [inline Rebracer.xml]: These are caused by an update to the Visual Studio extension Rebracer that switches code format settings for a solution based on this configuration file. These are just new configuration options that are supported and get added to the file automatically.

**Comment by CharliePoole** [comment]
Looks good to me.

File: src/NUnitFramework/tests/Attributes/TimeoutTests.cs [hunk 3]
```diff
@@ -131,6 +131,7 @@ public void TestTimeOutElapsed()
             Assert.That(result.ResultState, Is.EqualTo(ResultState.Failure));
             Assert.That(result.Message, Does.Contain("100ms"));
         }
+#if !NETCF
```
Comment (rprouse) [inline src/NUnitFramework/tests/Attributes/TimeoutTests.cs]: This test is failing on CF, we can investigate after the release. See #789

File: nuget/nunitlite.nuspec [hunk 4]
```diff
@@ -30,8 +30,8 @@
     <file src="bin/net-4.0/nunitlite.dll" target="lib/net40" />
     <file src="bin/net-4.5/nunitlite.dll" target="lib/net45" />
     <file src="bin/sl-5.0/nunitlite.dll" target="lib/sl5" />
-    <file src="../../nuget/Program.cs" target="content" />
-    <file src="../../nuget/Program.vb" target="content" />
-    <file src="../../nuget/install.ps1" target="tools" />
+    <file src="..\..\nuget\Program.cs" target="content" />
```
Comment (rprouse) [inline nuget/nunitlite.nuspec]: This fix for Linux didn't work on Windows. The three files were not included in the NuGet package and even worse, Program.cs was removed when I upgraded. This fixes it, but we will need to revisit #760 if packaging is important on Linux.

It looks like relative paths don't work with non-Windows separators, so we will likely need to copy these three files into the image directory.

**Comment by rprouse** [comment]
The release has been prepared and uploaded, merging so that I can create the GitHub release.

File: src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj (hunk 1 above)
Comment (CharliePoole) [inline src/NUnitEngine/nunit.engine.tests/nunit.engine.tests.csproj]: I guess we need separate package files for Linux then. I was used to the old NAnt scripts, where you just do everything using a forward slash. Apparently not so with MsBuild.

**Pull request merged**